semgrep_output_v1.atd

```diff
@@ -400,12 +408,14 @@ type incompatible_rule
    warning = something wrong that should be fixed
    info = some special condition worth knowing about
 *)
-type core_severity
-    <ocaml attr="deriving show">
+type severity
+    <ocaml attr="deriving show, eq">
     <python decorator="dataclass(frozen=True)"> = [
   | Error <json name="error">
```

Comment:
I think all of those need to be in uppercase, <json name="ERROR">
according to Cli_json_output.ml and the string_of_severity function

Reply:
Sure.